source/_docs/configuration/customizing-devices.markdown

```diff
@@ -19,8 +19,10 @@ To change the entity ID and friendly name of supported entities, follow these st
    ![Entity dialog box with cog icon.](/images/docs/configuration/customizing-entity-dialog.png)
 
 3. Enter the new name or the new entity ID.
-   - For example, the entity ID here could be `light.bedroom_ligthstrip`.
-   - Do not change the domain of the entity - the part before the `.`.
+   - For example, the entity ID here could be `light.bedroom_ligthstrip_3`.
```

Comment:
> Consider revising the example entity ID to avoid ending with an underscore.
> 
> ```diff
> -  light.bedroom_ligthstrip_3
> +  light.bedroom_lightstrip3
> ```
> 
> This change ensures the example adheres to the guidelines that entity IDs must not start or end with an underscore.
> 
> Committable suggestion
> Correct the typographical error in the entity ID.
> 
> ```diff
> -  light.bedroom_ligthstrip_3
> +  light.bedroom_lightstrip_3
> ```
> 
> This correction ensures the entity ID is spelled correctly, enhancing the professionalism of the documentation.
> 
> Committable suggestion

But it ends with a number, not with an underscore

Reply:
@c0ffeeca7 Its second suggestion, however, is correct

Reply:
> @c0ffeeca7 Its second suggestion, however, is correct

but that is not its suggestion. It's my original proposal. Which it first told me it was wrong.

Reply:
Its second suggestion reveals a typo...

![CleanShot 2024-05-24 at 22 20 47@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/a99df9d5-ec66-497f-8cb5-2270a23fb3dd)

`ligth` vs `light`